<!-- 
	This launch file should be run on the Jetson in order to start our drive, control, and auton
software
	on the physical rover.
 -->
<launch>
    <!-- launch rover core nodes -->
    <include file="$(find mrover)/launch/rover_core.launch" />

    <!-- params for hardware interfaces including IMU, GPS, and moteus -->
    <rosparam command="load" file="$(find mrover)/config/esw.yaml" />

    <!-- camera node -->
    <node name="arm_bridge" pkg="mrover" type="arm_bridge" />
</launch>